src/bvals/fc/bvals_shear_emf.cpp

```diff
@@ -9,6 +9,7 @@
 //========================================================================================
 
 // C headers
+#include <float.h>
```

Comment:
```suggestion
#include <float.h>
```

see below, use `#include <limits>` (or at a minimum, cfloat)